packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -296,7 +301,12 @@ export class SDKClient {
 
     return {
       fileId,
-      txResponse: await this.executeTransaction(ethereumTransaction, callerName, interactingEntity, requestId),
+      txResponse: (await this.executeTransaction(
+        ethereumTransaction,
+        callerName,
+        interactingEntity,
```

Comment:
Nit. Have you considered declaring `interactingEntity` as a `string` for type safety?